src/lib/serviceRoutinesV2/getRegistration.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+*
+* Copyright 2017 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Ken Zangelin
+*/
+#include <string>
+#include <vector>
+
+#include "common/statistics.h"
+#include "common/clockFunctions.h"
+#include "common/string.h"
+
+#include "rest/ConnectionInfo.h"
+#include "rest/OrionError.h"
+#include "ngsi/ParseData.h"
+#include "apiTypesV2/Registration.h"
+#include "mongoBackend/mongoRegistrationGet.h"
+#include "serviceRoutinesV2/getRegistration.h"
+#include "alarmMgr/alarmMgr.h"
+
+
+
+/* ****************************************************************************
+*
+* getRegistration - 
+*
+* GET /v2/registration/<reg id>
```

Comment:
GET /v2/registrations/<reg id>

(with 's' in the URL)

Reply:
Maybe you missed the "Fixed in" comment here?

Reply:
Fixed in f114c3e